relay-server/src/actors/project.rs

```diff
@@ -142,9 +142,15 @@ pub struct LimitedProjectConfig {
     pub allowed_domains: Vec<String>,
     pub trusted_relays: Vec<PublicKey>,
     pub pii_config: Option<PiiConfig>,
+    #[serde(skip_serializing_if = "DataScrubbingConfig::is_disabled")]
```

Comment:
there is a remote derive for ProjectConfig on this struct. Are you sure those things are needed here?

Reply:
nevermind, I _think_ this makes sense after re-reading how remote works

Reply:
I'm not sure, but it looks to me like an oversight. I will definitely add more tests at a later point.